src/settings/JobProfiles/resultsFormatter.js

```diff
@@ -28,21 +36,39 @@ export const resultsFormatter = (searchTerm, selectRecord, selectedRecords) => (
       />
     </div>
   ),
-  name: record => (
-    <AppIcon
-      size="small"
-      app="data-import"
-      iconKey="jobProfiles"
-      className={sharedCss.cellAppIcon}
-    >
-      <HighLight
-        search={searchTerm}
-        className={sharedCss.container}
+  name: record => {
+    const name = (
```

Comment:
Since there is no one for `nameLink` button, I do not see a need in the variable here.